### PR TITLE
Start setting up a pilot simulation driven by SOSE part 3

### PR DESCRIPTION
**Pull request by ali-ramadhan**

### CODE REVIEW

File: src/make_lesbrary_plots.py [hunk 1]
```diff
@@ -96,8 +96,64 @@ def plot_large_scale(ds):
...
+        uu.plot(ax=axes[1, 0], y="zC", label="$\overline{u^\prime u^\prime}$")
+        vv.plot(ax=axes[1, 0], y="zC", label="$\overline{v^\prime v^\prime}$")
+        ww.plot(ax=axes[1, 0], y="zC", label="$\overline{w^\prime w^\prime}$")
+        axes[1, 0].legend(loc="best", frameon=False)
+        axes[1, 0].set_xlabel("turbulent kinetic energy [m²/s²]")
+
+        uv = ds.uv.isel(time=n).squeeze()
+        uw = ds.uw.isel(time=n).squeeze()
+        vw = ds.vw.isel(time=n).squeeze()
+        uv.plot(ax=axes[1, 1], y="zC", label="$\overline{u^\prime v^\prime}$")
```
Comment (glwagner) [inline src/make_lesbrary_plots.py]: why `uv`?

File: src/sose_data.py [hunk 2]
```diff
@@ -91,6 +91,8 @@ def compute_geostrophic_velocities(ds, lat, lon, day_offset, days, zF, α, β, g
     }
 
     # xgcm grid for calculating derivatives and interpolating
+    # Not sure why it's periodic in Y but copied it from the xgcm SOSE example:
+    # https://pangeo.io/use_cases/physical-oceanography/SOSE.html#create-xgcm-grid
     grid = xgcm.Grid(ds, metrics=metrics, periodic=('X', 'Y'))
```
Comment (glwagner) [inline src/sose_data.py]: Perhaps this only matters if you differentiate across the boundaries of the domain.

File: src/make_lesbrary_plots.py (hunk 1 above)
Comment (glwagner) [reply src/make_lesbrary_plots.py]: We don't have a use for `uv` for single column stuff --- though its *possible* we could use it in simulations with lateral variability. It looked like you were plotting it so I wondered why.

File: src/pilot_simulation.jl [hunk 3]
```diff
@@ -279,23 +279,25 @@ slice_output_writer =
...
-    "κS" => model -> κSp(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "uu" => model ->  uu(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "vv" => model ->  vv(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "ww" => model ->  ww(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "uv" => model ->  uv(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "uw" => model ->  uw(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "vw" => model ->  vw(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "wT" => model ->  wT(model)[1+model.grid.Hz:end-model.grid.Hz],
-    "wS" => model ->  wS(model)[1+model.grid.Hz:end-model.grid.Hz]
+    "u"  => model ->  Up(model)[1+Hz:end-Hz],
```
Comment (glwagner) [inline src/pilot_simulation.jl]: Could be nice to save the full profiles including halo regions. I hope to have an LES model with diffusivities prescribed on boundaries, which means the gradients of fields across boundaries will be meaningful (and that information is encoded in halos).

File: src/make_lesbrary_plots.py (hunk 1 above)
Comment (ali-ramadhan) [reply src/make_lesbrary_plots.py]: Good points. Yeah was just plotting it to be comprehensive but guess it's not super useful for this pilot simulation.

File: src/pilot_simulation.jl (hunk 3 above)
Comment (ali-ramadhan) [reply src/pilot_simulation.jl]: True. Would be easy to add the option to `NetCDFOutputWriter`. I opened an issue about this: https://github.com/climate-machine/Oceananigans.jl/issues/683

**Pull request merged**